facebookads/api.py

```diff
@@ -393,7 +393,11 @@ def add(
             params = _top_level_param_json_encode(params)
             keyvals = ['%s=%s' % (key, urls.quote_with_encoding(value))
                        for key, value in params.items()]
-            call['body'] = '&'.join(keyvals)
+            payload = '&'.join(keyvals)
+            if method is 'POST' or method is 'PUT':
```

Comment:
String comparison should use `==` and not `is`, see for example http://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce

You can also express the same thing with
```
if method in ('POST', 'PUT'):
```